UnityProject/Assets/Scripts/Clothing/FacehuggerImpregnation.cs

```diff
@@ -37,6 +37,14 @@ private void Awake()
 
 		public override void OnStartServer()
 		{
+			// Testing to see if one of the several references to this method get called on the client. Can be removed
+			// or add a [Server] line above it fire warning, but this is more specific at a glance.
+			if (!isServer) 
```

Comment:
OnStartServer is only ever called on the server or a host so `!isServer` is not needed here

Reply:
I agree. Early on I started with a shotgun approach and simply tossed isServer checks around in various places with a log to look out for once I started the client. I planned to clean up the ones that never showed up and then incrementally remove the ones that did get run by the client somehow after I figured out how/why they were being called.